frontend/src/pages/hardwareProfiles/utils.ts

```diff
@@ -152,12 +152,10 @@ export const validateProfileWarning = (
   const warnings = hardwareProfileWarningSchema.safeParse({ isDefault: false, value: identifiers });
```

Comment:
lets remove this isDefault becuase that is deprecated logic now. We dont have "default" hardware profiles anymore